src/pages/manager/CustomersListPage.js

```diff
@@ -2,10 +2,56 @@ const { expect } = require('@playwright/test');
 
 export class CustomersListPage {
   constructor(page) {
-    this.page = page; 
+    this.page = page;
+    this.lastRow = page.getByRole('row').last();
+    this.accountNumberCell = this.lastRow.getByRole('cell').nth(3);
+    this.searchField = this.page.getByPlaceholder('Search Customer');
+  
+    this.firstRow = page.getByRole('row').nth(1);
+    this.firstNameCell = this.firstRow.getByRole('cell').nth(0);
+
+    this.lastNameCell = this.firstRow.getByRole('cell').nth(1);
+
+    this.postCodeCell = this.firstRow.getByRole('cell').nth(2);
```

Comment:
The use of `firstRow` to access the first name, last name, and post code cells assumes that the first row is always the correct one to check. This might not be reliable if the table is sorted or filtered. Consider implementing a more robust way to verify the customer details.